src/lib.rs

```diff
@@ -867,6 +868,42 @@
 //! }
 //! ```
 //!
+//! ### Flattening subcommands
+//!
+//! It is also possible to combine multiple enums of subcommands into one.
+//! All the subcommands will be on the same level.
+//!
+//! ```
+//! # use structopt::StructOpt;
+//! # fn main() {}
+//! #[derive(StructOpt)]
+//! enum BaseCli {
+//!     Ghost10(Ghost10),
+//! }
+//!
+//! #[derive(StructOpt)]
+//! struct Ghost10 {
+//!     arg1: i32,
+//! }
```

Comment:
You can use "enum struct". It's quite convenient in examples as it is shorter.

```suggestion
//! enum BaseCli {
//!     Ghost10 { arg1: i32 },
//! }
```